application/pom.xml

```diff
@@ -89,5 +89,10 @@
       <artifactId>mockito-core</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>io.dropwizard</groupId>
+      <artifactId>dropwizard-auth</artifactId>
+      <version>${dropwizard.version}</version>
```

Comment:
You don't need the `<version>` tag in this pom at all, it inherits the version from the parent pom.